packages/react-generator/src/templates/react-template/template-files/src/http/axios.js

```
@@ -0,0 +1,139 @@
+      }
+
+      if (Array.isArray(config)) {
+        config.forEach((item) => {
+          mock.proxy(item)
+        })
+      }
+
+      return mock
+    },
+    disableMock() {
+      mock && mock.restore()
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)